mappings/net/minecraft/world/attribute/EnvironmentAttributeMap.mapping

```diff
@@ -11,35 +11,51 @@ CLASS net/minecraft/class_12199 net/minecraft/world/attribute/EnvironmentAttribu
 	METHOD method_75661 builder ()Lnet/minecraft/class_12199$class_12200;
 	METHOD method_75662 getEntry (Lnet/minecraft/class_12197;)Lnet/minecraft/class_12199$class_12201;
 		ARG 1 key
-	METHOD method_75663 get (Lnet/minecraft/class_12197;Ljava/lang/Object;)Ljava/lang/Object;
+	METHOD method_75663 apply (Lnet/minecraft/class_12197;Ljava/lang/Object;)Ljava/lang/Object;
```

Comment:
The map doesn't hold the return value, and its name was too confusing with other getOrDefault impls.